escrow/payment_channel_storage_test.go

```diff
@@ -145,10 +149,21 @@ func (suite *BlockchainChannelReaderSuite) TestGetChannelState() {
 func (suite *BlockchainChannelReaderSuite) TestGetChannelStateIncorrectGroupId() {
 	reader := suite.reader
 	reader.replicaGroupID = func() ([32]byte, error) { return [32]byte{32}, nil }
+	reader.recipientPaymentAddress = func() common.Address { return suite.recipientAddress }
 
 	channel, ok, err := reader.GetChannelStateFromBlockchain(suite.channelKey())
 
 	assert.Equal(suite.T(), errors.New("Channel received belongs to another group of replicas, current group: [32 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0], channel group: [123 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]"), err)
 	assert.False(suite.T(), ok)
 	assert.Nil(suite.T(), channel)
 }
+
+func (suite *BlockchainChannelReaderSuite) TestGetChannelStateIncorrectRecipeintAddress() {
+	reader := suite.reader
+	reader.replicaGroupID = func() ([32]byte, error) { return [32]byte{123}, nil }
```

Comment:
Was done to prevent clash with groupID check

Reply:
Ok, its minor anyway but I mean that groupID check should pass because default blockchain reader you copied here returns the same groupId (see line 18).